### DISCUSSION TITLE
Tokens Studio breaks styles in the Prime Figma Toolkit

### DISCUSSION THREAD

**Discussion by rtkramer** [primefaces/community#2606] [Figma UI Kit]
When updating a change to a component using the Tokens Studio Plugin, styles break and become default values, #hex codes for colors, raw text (Roboto, Inter etc.) values for type layers. This is a breaking change to the Prime Figma 2.2.0 Toolkit and Tokens Studio workflow. I need to re-map all of my styles to every component in the kit, no big deal. Just wondering if anyone noticed this. I will look for a thread, I forgot to check first. This just broke my entire workflow yesterday, seems like the styles are there, so styles can be used by designers for new content. Also dev mode still shows the token selections. So this only seems to affect the style connection. but my designers cannot click on layers on instances from the Prime library and see which styles are applied.

See image for what I see in my Figma
![Token Studio](https://github.com/user-attachments/assets/ec9407f7-b239-482e-ab65-6f1412585803)

**Comment by onursenture** [comment]
Hello, we are retrospectively reviewing issues in Discussions. We apologize for not responding to your question promptly. The token system has changed with the introduction of UI Kit 3. Since there are no typography tokens in our current system, we cannot reproduce the issue to determine if it persists.
